src/main/java/techreborn/client/RegisterItemJsons.java

```diff
@@ -131,11 +133,10 @@ private static void registerItems() {
 			registerBlockstate(ModItems.GEMS, i, name[i], "items/materials/");
 		}
 		
-		name = ItemPlates.types.clone();
-		for (int i = 0; i < ItemPlates.types.length; ++i) {
-			registerBlockstate(ModItems.PLATES, i, name[i], "items/materials/");
+		for (ModPlates plate : ModPlates.values()){
```

Comment:
Move to a SideOnly(Client) method in the enum, same as registering the items.

Reply:
ok